webapp/src/main/java/org/apache/atlas/web/rest/AuthREST.java

```diff
@@ -164,6 +172,47 @@ public ServicePolicies downloadPolicies(@PathParam("serviceName") final String s
         }
     }
 
+    @POST
+    @Path("refreshcache")
```

Comment:
can we rename this to `refreshauthcache` or something else, which resembles we are refreshing Auth relates cache